src/scram.c

```diff
@@ -1081,8 +1087,8 @@ scram_verify_plain_password(PgSocket *client,
 		password = prep_password;
 
 	/* Compute Server Key based on the user-supplied plaintext password */
-	scram_SaltedPassword(password, salt, saltlen, iterations, salted_password);
-	scram_ServerKey(salted_password, computed_key);
+	scram_SaltedPassword(password, salt, saltlen, iterations, salted_password, &errstr);
+	scram_ServerKey(salted_password, computed_key, &errstr);
```

Comment:
Shouldn't we be doing something with errstr?

Reply:
Yes. I forgot to add a TODO. A few error/warning messages was added to these `scram_XXX` functions in Postgres and we should do the same in PgBouncer as well.